src/cli.ts

```diff
@@ -36,12 +37,14 @@ function readCommandFlags(resolveRootRelative: (p: string) => string): Options {
     W: skipWithInit = false,
     n: namespaced = false,
     N: namespaceExclude = [],
+    P: peerProject = false,
```

Comment:
Should we be woried about `p` and `P` both being used for unrelated things?
If yes, what letter should we use here ?

Reply:
Yeah I would try to use a different letter. We can use `q` perhaps?